client/options.go

```diff
@@ -41,6 +41,9 @@ type Options struct {
 	// Other options for implementations of the interface
 	// can be stored in a context
 	Context context.Context
+
+	// ProxyAddress to send requests via
+	ProxyAddress string
```

Comment:
Just cal Proxy and comment // Address to proxy all requests through

Reply:
why not say Address and comment about that this is endpoint address or some proxy?